FileIO/GMSInterface.cpp

```diff
@@ -240,6 +240,7 @@ MeshLib::Mesh* GMSInterface::readGMS3DMMesh(const std::string &filename)
 	INFO("GMSInterface::readGMS3DMMesh(): Read GMS-3DM mesh.");
 	std::vector<MeshLib::Node*> nodes;
 	std::vector<MeshLib::Element*> elements;
+	std::vector<int> mat_ids;
```

Comment:
`mat_id` below is declared as `unsigned` but should be consistent with `mat_ids` vector type.